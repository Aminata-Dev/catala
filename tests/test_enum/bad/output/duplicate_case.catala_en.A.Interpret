[ERROR] The constructor Case3 has been matched twice:

  --> tests/test_enum/bad/duplicate_case.catala_en
   | 
18 |     -- Case3 : true
   |                ^^^^
   + Article

  --> tests/test_enum/bad/duplicate_case.catala_en
   | 
17 |     -- Case3 : false
   |                ^^^^^
   + Article
