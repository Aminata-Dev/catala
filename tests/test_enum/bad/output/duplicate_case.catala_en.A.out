[ERROR] The constructor Case3 has been matched twice:

  --> test_enum/bad/duplicate_case.catala_en
   | 
18 |     -- Case3 : true
   |                ^^^^
   + Article

  --> test_enum/bad/duplicate_case.catala_en
   | 
17 |     -- Case3 : false
   |                ^^^^^
   + Article
